tests/acceptance/bootstrap/WebDav.php

```diff
@@ -1677,7 +1677,7 @@ public function userUploadsAFileToUsingWebDavApi(
 	public function userHasUploadedAFileTo(string $user, string $source, string $destination):array {
 		$response = $this->uploadFile($user, $source, $destination, null, true);
 		$this->theHTTPStatusCodeShouldBe(
-			["201", "204"],
+			["201", "204", "200"],
```

Comment:
why `200`? which step needs it?

Reply:
template uploads

Reply:
It seems size matters 200 if big size file